js/app.js

```diff
@@ -0,0 +1,277 @@
+// jQuery extension to handle css animations
+
+$.fn.extend({
+    animateCss: function(animationName) {
+        var animationEnd = 'webkitAnimationEnd mozAnimationEnd MSAnimationEnd oanimationend animationend';
+        $(this).addClass('animated ' + animationName).one(animationEnd, function() {
+            $(this).removeClass('animated ' + animationName);
+        });
+    }
+});
+
+
+// Pirate Bar Sounds // 
+
+var tavern = new Audio('sounds/Medieval_tavern.mp3');
+var yar = new Audio('sounds/yaargh.wav');
+var drivesMe = new Audio('sounds/drivesmenuts.wav');
+var shiver = new Audio('sounds/shivermetimbers.wav');
+var ahoy = new Audio('sounds/ahoymatey.wav');
+var lillylivered = new Audio('sounds/lillyliveredlandlubbers.wav');
+var plank = new Audio('sounds/walktheplank.wav');
+
+
+// Constructor Functions // 
+
+var PirateQuestions = function(questions, flavor) {
+    this.questions = questions;
+    this.flavor = flavor;
+
+};
+
+
+var PirateIngredients = function(ingredients, amounts) {
+    this.ingredients = ingredients;
+    this.amounts;
+};
+
+
+var PirateBartender = function(questions) {
+    this.questions = questions;
+    this.userPreference = [];
+}
+
+
+
+PirateQuestions.prototype.display = function() {   
+    $('.pirate-questions').text(' ');
+    $('.pirate-questions').animateCss('zoomIn');
+    $('.pirate-questions').append('<h2>' + this.questions + '</h2>');
+
+};
+
+
+// store random number and use same random number in splice()
+// make var for holding random num 
+
+
+
+PirateBartender.prototype.checkUserAnswerYes = function() {
+	if (this.questions.length < 0) {
+
+	} else {
+        this.userPreference.push(this.currentQuestion.flavor);
+        console.log(this.userPreference + " pushed");
+        this.checkQuestions();
+    }
+    
+};
+
+PirateBartender.prototype.checkUserAnswerNo = function() {	
+	this.checkQuestions();
+	
+};
+
+PirateBartender.prototype.checkQuestions = function() {
+    if (this.questions.length == 0) {        
+        this.makeDrink();
+        
+    } else {
+        var randomNum = Math.floor(Math.random() * questions.length);
+        this.currentQuestion = this.questions[randomNum];
+        this.questions.splice(randomNum, 1);
+        this.currentQuestion.display();
+    }
+};
+
+
+PirateBartender.prototype.makeDrink = function() {    
+    for (var i = 0; i < this.userPreference.length; i++) {
+        var flavor = this.userPreference[i];
+        var flavorIngredients = Ingredients[flavor];
+        var randomNum = Math.floor(Math.random() * flavorIngredients.length);
+        var randomIngredient = flavorIngredients[randomNum];
+        $('.pirate-questions').text(' ');
+        $('.pirate-questions').append('<h2>' + "Here's yer " + randomIngredient + '</h2>');
```

Comment:
should be consistent with quote usage. It stands out when you have ' asdfasd' + " asdfasdf "